marketplace-service/src/main/java/com/axonivy/market/service/impl/ProductServiceImpl.java

```diff
@@ -113,28 +102,25 @@ public class ProductServiceImpl implements ProductService {
   private final ProductJsonContentRepository productJsonContentRepo;
   private final ImageRepository imageRepo;
   private final ImageService imageService;
-  private final MongoTemplate mongoTemplate;
   private final ProductContentService productContentService;
   private final ExternalDocumentService externalDocumentService;
-  private final ObjectMapper mapper = new ObjectMapper();
-  private final SecureRandom random = new SecureRandom();
   private final MetadataService metadataService;
+  private final ProductMarketplaceDataService productMarketplaceDataService;
   private GHCommit lastGHCommit;
   private GitHubRepoMeta marketRepoMeta;
   @Value("${market.legacy.installation.counts.path}")
   private String legacyInstallationCountPath;
```

Comment:
No more usage, please remove it

Reply:
I use productMarketplaceDataService in some fetch product detail methods

Reply:
![image](https://github.com/user-attachments/assets/e9a76598-f16d-4f42-8db6-d2a5af73a910)
I'm not sure abt that when ide showed me no usage